_data/yashtandon019/yashtandon019.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Yash Tandon
+institution: SRM Institute Of Science And Technology, Chennai, Tamil Nadu, India
```

Comment:
The name of the institution must **not** be longer than 58 characters, yours is **67** characters.